apps/src/code-studio/components/backpack/BackpackClientApi.js

```diff
@@ -21,6 +25,38 @@ export default class BackpackClientApi {
     });
   }
 
+  fetchFile(filename, onError, onSuccess) {
+    if (!this.channelId) {
```

Comment:
nit: potentially just use `!this.hasBackpack()` for these checks? So that logic can stay contained to `hasBackpack` if it ever becomes more than just checking channel ID?

Reply:
good call, done!